platform/android/app/src/main/res/font

```diff
@@ -0,0 +1 @@
+../../../../../../assets/fonts
```

Comment:
OK, this is how a moved symbolic link appears, right?

Reply:
Yup!